numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1827,13 +1863,13 @@ unpack_bits(PyObject *input, int axis, PyObject *count_obj)
         if (out_stride == 1) {
             /* for unity stride we can just copy out of the lookup table */
             for (index = 0; index < in_n; index++) {
-                memcpy(outptr, unpack_lookup[*inptr], 8);
+                memcpy(outptr, unpack_lookup[0][*inptr], 8);
```

Comment:
nit: `(*unpack_lookup)[*inptr]` would be clearer, semantically, `unpack_lookup` is not an array at all.

Reply:
adopted

Reply:
removed all this